test.sh

```diff
@@ -146,7 +146,18 @@ GOBIN=${GOBIN:-$HOME/gopath/bin}
 #
 if [[ "$RUN" =~ "vet" ]] ; then
   start_context "test/vet"
-  run_and_comment go vet ./...
+  check_go_vet() {
```

Comment:
This is unnecessary. `run_and_comment` already collects the output of the command in `${result}`. Currently it only comments if the status code is > 0. With your lint fixes, you should update run_and_comment so it sends a comment if `${result}` is non-empty. I took a look through the other run_and_comment invocations and I believe they all have the same semantics: no output on success, output on failure.